internal/daysteps/daysteps.go

```diff
@@ -1,13 +1,76 @@
 package daysteps
 
+import (
+	"fmt"
+
+	"github.com/Yandex-Practicum/tracker/internal/personaldata"
+
+	// "internal/personaldata"
+	"github.com/Yandex-Practicum/tracker/internal/spentenergy"
+	//"internal/spentenergy"
+	"strconv"
+	"strings"
+	"time"
+)
+
 type DaySteps struct {
-	// TODO: добавить поля
+	Steps    int
+	Duration time.Duration
+	personaldata.Personal
 }
 
 func (ds *DaySteps) Parse(datastring string) (err error) {
-	// TODO: реализовать функцию
+	// 1. разделить строку на слайс строк
+	slice := strings.Split(datastring, ",")
+
+	// 2. проверить, чтобы длина слайса была равна 2, так как в строке данных у нас количество шагов и продолжительность
+	if len(slice) != 2 {
+		return fmt.Errorf("длина слайса не равна 2")
+	}
+
+	// 3. преобразовать первый элемент слайса (количество шагов) в тип int. Обработать возможные ошибки. При их возникновении
+	// из функции вернуть 0 шагов, 0 продолжительность и ошибку
+	steps, err := strconv.Atoi(slice[0])
+	if err != nil {
+		return fmt.Errorf("не удалось преобразовать количество шагов: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return fmt.Errorf("не удалось преобразовать количество шагов: %w", err)
```

Reply:
Нужно везде поправить